frontend/uddhar-frontend/src/shared/components/AlertDialog.jsx

```
@@ -27,24 +27,24 @@ const AlertDialog = ({ setOnConfirm, setShowAlert, message }) => {
           Are you absolutely sure?
         </h2>
-        <p className="text-[#9ca3af] mb-6">
+        <p className="text-black mb-6">
           {message || "This action cannot be undone. This will permanently stop the event."}
         </p>
         <div className="flex justify-end gap-3">
           <button
             onClick={onClickHandler}
-            className="px-4 py-2 rounded-md bg-transparent text-white hover:bg-gray-700 transition-colors cursor-pointer"
+            className="px-4 py-2 rounded-md bg-transparent text-black hover:bg-gray-300 transition-colors cursor-pointer"
           >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)